### ISSUE TITLE
Pre-trained waveglow not producing results like samples

### ISSUE THREAD

**Issue by markopolojarvi** [NVIDIA/tacotron2#145]
I've been playing around with tacotron and it's such a great piece of work but I'm guessing I'm missing something because I can't get it to produce similar results as demonstrated with https://github.com/NVIDIA/waveglow.

I'm using pre-trained waveglow_old for testing but I'm getting a lot of artifacts.

Here's a sample of what I'm getting https://picosong.com/wJMxP/

**Comment by rafaelvalle** [comment]
What parameter sigma did you use to produce this sample?

**Comment by markopolojarvi** [comment]
@rafaelvalle 0.666. It's pretty much exactly as in https://github.com/NVIDIA/tacotron2/blob/master/inference.ipynb.

**Comment by rafaelvalle** [comment]
What's your Taco2 val loss?

**Comment by markopolojarvi** [comment]
I know this is incredibly unhelpful but I don't know because I submitted this 8 days ago and colab resets itself day after so I lost everything I had done.

However I didn't change anything from inference.ipynb at the end and I was still getting the result I linked above. So any ideas how I could improve the results? I looked at hparams but without understand the code, the options are a bit of blackbox knobs ;)
